server/models/service-instance.js

```diff
@@ -321,6 +321,23 @@ module.exports = function extendServiceInstance(ServiceInstance) {
   }
   ServiceInstance.prototype.logDump = logDump;
 
+  [
+    'agentTraces', 'expressUsageRecords', 'serviceMetrics',
+    'getMetaTransactions', 'getTransaction', 'getTimeline',
+    'getTrace', 'stopObjectTracking', 'startObjectTracking',
+    'startCpuProfiling', 'stopCpuProfiling', 'heapSnapshot',
+    'queryCapabilities', 'queryCapabilitiesAll', 'applyPatch'
+  ].forEach(function(methodName) {
+    ServiceInstance.prototype[methodName] = function(pk) {
```

Comment:
I'm a fan of metaprogramming in general, but I don't think it buys us much in this case since the argument massaging to account for differences ends up eating the difference.

Maybe instead of a loop, write a 1 arg and 2 arg variant of a helper and just call it for each one?

```js
proxy2(ServiceInstance, 'processes', 'getTrace');
proxy1(ServiceInstance, 'processes', 'getTimeline');
// ...
```